static/js/muting.js

```diff
@@ -36,7 +36,7 @@ export function is_topic_muted(stream_id, topic) {
         return false;
     }
     const sub_dict = muted_topics.get(stream_id);
-    return sub_dict && sub_dict.get(topic);
+    return (sub_dict && sub_dict.get(topic)) || false;
```

Comment:
Isn't this redundant code? `flag OR false` is identically equal to `flag`.

Reply:
The only possible cases here are:

1. `undefined` || `false` => `false`
2. `some integer` || `false` => `integer itself`.
(here it can also be `true` || `false` => `true`).

Reply:
Is the intent to convert the return value to a Boolean? Maybe `return (sub_dict && sub_dict.get(topic)) !== undefined` might be cleaner then?